### DISCUSSION TITLE
Badge in masthead

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#2230] [Q&A] [answered]
We are trying to get users to reverify their email addresses and are contemplating different solutions. One solution is to use an alert on the Console dashboard but the Console team is concerned about "banner blindness" created by overusage of banners. Another exploration I have is using a [popover badge](https://paste.twilio.design/components/popover/#popoverbadgebutton) or a tooltip. I was wondering if you had any thoughts and preferences on any of these explorations? You can see the explorations here:
https://www.figma.com/file/G33XZnaK8NeIATwRsi5ptJ/?node-id=503%3A33407

**Comment by TheSisb** [comment, marked as answer]
Hello @pguptaDS,

This is a difficult problem to solve from a design system point of view. You may want to check with the Console team on how they define global error / warning messaging like these should work. As you may know, the header already has a trial badge (image attached), so one-off solutions may not scale correctly if both happen to be visible at once. 

![image](https://user-images.githubusercontent.com/1592327/155218385-7f24d464-403d-4371-bc09-455bf2306b9b.png)

In an ideal world, you would have definitions of varying notification severity and recommended treatments for each. This seems relatively low severity (just an assumption) vs. [the A2P message from this older discussion](https://github.com/twilio-labs/paste/discussions/2186), so the design treatment should reflect that.  

Hope this helps!

**Reply by pguptaDS** [reply]
Thanks for the response, Shadi! Had a lot of discussions with the console and the growth team to discuss a path forward here.

Quick question for you - Would it be ok to bring up a popover instead of a tooltip when the user clicks on the icon[ in this screen](https://www.figma.com/file/G33XZnaK8NeIATwRsi5ptJ/Miscellaneous?node-id=503%3A41182)? @TheSisb 

Tooltip disappears if you're not hovering over the icon and it wouldn't be possible to add a link within the tooltip so we would prefer the popover if possible.

**Reply by shleewhite** [reply]
Hi! A popover is totally possible, we have examples of badges with popovers on the docs site: https://paste.twilio.design/components/popover#popoverbadgebutton. This is a great use case for a popover instead of a tooltip.
